Makefile

```diff
@@ -89,8 +89,8 @@ test-builtin:
 	PL_DEVICE=$(if $(device:-=),$(device),lightning.qubit) $(PYTHON) -I $(TESTRUNNER)
 
 test-suite:
-	pl-device-test --device $(if $(device:-=),$(device),lightning.qubit) --skip-ops --shots=20000
-	pl-device-test --device $(if $(device:-=),$(device),lightning.qubit) --shots=None --skip-ops
+	pl-device-test --device $(if $(device:-=),$(device),lightning.qubit) --skip-ops --shots=20000 -n auto
+	pl-device-test --device $(if $(device:-=),$(device),lightning.qubit) --shots=None --skip-ops -n auto
```

Comment:
`test-suite` is quite fast, so I don't know whether it is worth forcing a dependency on `pytest-xdist`, but I don't mind either way.